src/components/Home/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { SecretDataType } from "../../secretDataType";
+
+export default function Home(props: { showSecretData: Array<SecretDataType> }) {
```

Comment:
SecretDataType[]

Reply:
showSecretData seem like a function name. But it's actually an array. You'd better use `secretDataList`